scripts/buildkite/build_tpp.sh

```diff
@@ -33,10 +33,10 @@ fi
 if [ "${COMPILER}" == "clang" ]; then
   GCC_COMPAT_OPTION="-g ${GCC_TOOLCHAIN}"
 fi
-if [ "${SANITIZERS}" ]; then
+if [ "${SANITIZERS}" ] && [ "0" != "${SANITIZERS}" ]; then
```

Comment:
I'd just make:
```
if [ "1" == "${SANITIZERS}" ]; then
```

Reply:
Yeah, then SANITIZER=2 would not match although unequal zero. Anyway, the current pattern is used elsewhere also.

Reply:
Who would ever use `=2`? I'm trying to simplify the pattern elsewhere too.

Reply:
Completed.